### ISSUE TITLE
Can I implement it on tableview

### ISSUE THREAD

**Issue by jassimkm** [ConfianceLabs/CLTimer#2]
Hi, 
Is there is any way to add CLtimer on tableview cells. I want to hide cltimer when timer is stopped and show another view in the cell . Is that possible 
Thanks and Regards

**Comment by ConfianceLabs** [comment]
Yes of course it's possible simply take a UIView in your tableview cell and
replace it's class with CLTimer
Step 2 create it's outlet
Step 3 take a unbutton on its action call stopTimer func of CLTimer and
then simply hide cltimer and bring your another view in scene by unhiding
it .
If you still didn't get it then let me know I will provide a sample project
to you .

**Comment by jassimkm** [comment]
Thanks , I got the point but I want to reload the cell when timer hits zero. I cant use your delegate . The app is crashing when I set . cell.timerView.cltimerDelegate = self
